changelogs/unreleased/gh-8613-new-cli-options.md

```diff
@@ -0,0 +1,3 @@
+## feature/cli
```

Comment:
Typo in commit message: "patch" -> "path"

Reply:
Fixed, thanks!